main.c

```diff
@@ -103,6 +103,8 @@ static void swayidle_finish() {
 	wl_list_for_each_safe(cmd, tmp, &state.timeout_cmds, link) {
 		wl_list_init(&cmd->link);
```

Comment:
Unrelated: this is broken, it overwrites the previous `link`. The `link` is already initialized. We should only call `wl_list_remove`, not `wl_list_init`.

Reply:
Opened #94 to remove that call.